web/src/features/fbaCalculator/components/FBATable.tsx

```diff
@@ -411,271 +412,282 @@ const FBATable = (props: FBAInputGridProps) => {
                   setSelected={setSelected}
                   loading={loading}
                 />
-                <TableBody data-testid="fba-table-body">
-                  {rows.map(row => {
-                    return (
-                      !isUndefined(row) && (
-                        <TableRow key={row.id}>
-                          <StickyCell left={0} zIndexOffset={1} backgroundColor="#FFFFFF">
-                            <SelectionCell
-                              selected={selected}
-                              updateSelected={(newSelected: number[]) =>
-                                setSelected(newSelected)
-                              }
-                              disabled={
-                                rowIdsToUpdate.has(row.id) && !rowShouldUpdate(row)
-                              }
-                              rowId={row.id}
-                            />
-                          </StickyCell>
-                          <LoadingIndicatorCell
-                            loading={loading}
-                            rowUpdating={rowIdsToUpdate.has(row.id)}
-                            initialLoad={initialLoad}
-                          >
+                {rows.length === 0 ? null : (
```

Comment:
Instead of returning `null` here you can return your new component. Then you won't need any more code below.